### PR TITLE
Add SBS PBR material data, more

### PR DESCRIPTION
**Pull request by luisbarrancos**
NEtemplates, documentation, screenshots

### CODE REVIEW

**Comment by luisbarrancos** [comment]
Test page for the branch in https://appleseed-maya2.readthedocs.io/en/sbs/shaders/material/as_sbs_pbrmaterial.html#label-as-sbs-pbrmaterial
You might need to login to RTD though. The git hash in the page is replaced by a version from the git tag, so ignore that hash for now.

File: docs/source/shaders/material/as_sbs_pbrmaterial.rst [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+
+.. [#] By default, the color channel, and other color quantities are set to be encoded with the sRGB OETF. But note that you cannot specify RGB primaries or white point in Substance Painter, so you should assume these to be the sRGB/Rec.709 RGB primaries and D65 whitepoint if you intend to convert to other color spaces.
+
+.. [#] Which in this case is the `Lambert BRDF <https://en.wikipedia.org/wiki/Lambertian>`_.
+   
+.. [#] See also `the definition of BSDF <https://en.wikipedia.org/wiki/Bidirectional_scattering_distribution_function>`_ for more details.
+
+.. [#] The *real* and absolute index of refraction of the material. Since dielectrics have a very small extinction coefficient, this is assumed to be 0, and monochromatic. Absolute since it's assumed to be the ratio of the wave in the medium against the ratio of the wave in air, which has a index of refraction very close to 1.0, so it's rounded to 1.0 (i.e, vacuum). Appleseed is **not** querying the exterior medium with *nested dielectrics* :cite:`ND.2002.10487555` to compute the correct index of refraction. The priority was given to have as much as possible a matching appearance with Substance Painter's PBR material.
```
[the remark this answers is not in the document]
Comment (luisbarrancos) [reply docs/source/shaders/material/as_sbs_pbrmaterial.rst]: Ratio of wave speed in vacuum over wave speed in medium. Rephrasing this.

File: docs/source/shaders/material/as_sbs_pbrmaterial.rst [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+*Roughness*
+    The roughness texture channel. A *raw* quantity usually exported as a grayscale texture.
+
+-----
+
+Opacity
+^^^^^^^
+
+*Opacity*
+    The opacity texture channel, with a value of 0.0 denoting a fully transparent surface, and a value of 1.0 denoting a fully opaque surface. A *raw* quantity usually exported as a greyscale texture.
```
Comment (dictoon) [inline docs/source/shaders/material/as_sbs_pbrmaterial.rst]: *grayscale* (fix in many other places)

File: docs/source/shaders/material/as_sbs_pbrmaterial.rst [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+   :title:
+
+   Greenish paint layers over smooth steel, with mesh curvature controlling the exposure of the underlying metallic substrate.
+
+.. thumbnail:: /_images/screenshots/sbs_pbrmaterial/sbs_pbr_mat4_evening.jpg
+   :group: shots_sbs_pbrmaterial_group_A
+   :width: 10%
+   :title:
+
+   Flaked orange paint over aluminium, with a layer of accumulated dirt on top. Curvature and grunge maps controlled the flakiness and erosion of the paint layer.
```
Comment (dictoon) [inline docs/source/shaders/material/as_sbs_pbrmaterial.rst]: It's **aluminum**.

File: docs/source/shaders/material/as_sbs_pbrmaterial.rst [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+
+.. rubric:: Footnotes
+
+.. [#] By default, the color channel, and other color quantities are set to be encoded with the sRGB OETF. But note that you cannot specify RGB primaries or white point in Substance Painter, so you should assume these to be the sRGB/Rec.709 RGB primaries and D65 whitepoint if you intend to convert to other color spaces.
+
+.. [#] Which in this case is the `Lambert BRDF <https://en.wikipedia.org/wiki/Lambertian>`_.
+   
+.. [#] See also `the definition of BSDF <https://en.wikipedia.org/wiki/Bidirectional_scattering_distribution_function>`_ for more details.
+
+.. [#] The *real* and absolute index of refraction of the material. Since dielectrics have a very small extinction coefficient, this is assumed to be 0, and monochromatic. Absolute since it's assumed to be the ratio of the wave speed vacuum and wave speed in the medium. Appleseed is **not** querying the exterior medium with *nested dielectrics* :cite:`ND.2002.10487555` to compute the correct index of refraction. The priority was given to have as much as possible a matching appearance with Substance Painter's PBR material.
```
Comment (dictoon) [inline docs/source/shaders/material/as_sbs_pbrmaterial.rst]: *the wave speed **in (the)** vacuum*

**Review by dictoon — CHANGES_REQUESTED** [review]

**Review by dictoon — APPROVED** [review]

**Pull request merged**